### PR TITLE
1.2.6 post

### PR DESCRIPTION
**Pull request by sargonas**

### CODE REVIEW

File: collections/_posts/2021-09-10-important-version-1.2.6-information.en.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## So what _is_ a Bluebox Timelord?
+
+As you may be aware, a [Timelord](https://github.com/Chia-Network/chia-blockchain/wiki/Timelords) is a server that generates the next block in the chain, with the fastest running one at any given moment being the authoritative server moving the chain forward. The Timelord infuses a winning proof of space into the blockchain, and then the process repeats. In order to do its job and do it well, a Timelord must be _fast_, at the expense of many other things including efficiency. This means the blocks it generates are uncompressed and ugly in the interest of speed. Enter the Bluebox… there can be any number of Blueboxes running, and their job is to scour the blockchain db and look for old blocks that are still uncompressed. They then clean up and compress a block, and then gossip it around to all of those nodes, who take this compressed block and perform a few confirmations, and then swap out the old bloated blocks in their DB for the new streamlined one.
```
Comment (mariano54) [inline collections/_posts/2021-09-10-important-version-1.2.6-information.en.md:21]: Timelords don't generate the next block, they generate proofs of time, which are a required part of the block.

Comment (mariano54) [reply collections/_posts/2021-09-10-important-version-1.2.6-information.en.md:21]: A block is a data structure which incldues a bunch of things:
- Proof of space
- Proof of time (VDF)
- Signature
- Transactions

The timelord only generates one piece

Comment (mariano54) [reply collections/_posts/2021-09-10-important-version-1.2.6-information.en.md:21]: Proof of time are large and take up a lot of space. Blueboxes look for large proofs of time and compress them.

Comment (mariano54) [reply collections/_posts/2021-09-10-important-version-1.2.6-information.en.md:21]: "perform a few confirmations", should be changed to: "verify the proof of time".
Confirmations means something else in blockchain (this transaction has 5 confirmations means that there were 5 blocks mined after that transactino was included).

**Review by mariano54 — COMMENTED** [review]
Some small nits on the technical language

File: collections/_posts/2021-09-10-important-version-1.2.6-information.en.md (hunk 1 above)
Comment (sargonas) [reply collections/_posts/2021-09-10-important-version-1.2.6-information.en.md]: updated!

**Review by hoffmang9 — APPROVED** [review]
ok

**Pull request merged**